jina/types/arrays/abstract.py

```diff
@@ -21,7 +21,7 @@ def get_attributes(self, *fields: str) -> Union[List, List[List]]:
 
         :param fields: Variable length argument with the name of the fields to extract
         """
-        pass
```

Comment:
declaring as `Abstractmethod` is enough u just need to do

```python
   ...
```

Reply:
okay, I'll see if it's excluded from codecov

Reply:
Done